docs/DeveloperGuide.md

```diff
@@ -302,49 +302,59 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `MedBook` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a patient**
+**Use case: UC01 - Viewing help**
 
 **MSS**
 
-1.  User requests to list patients
-2.  MedBook shows a list of patients
-3.  User requests to delete a specific patient in the list
-4.  MedBook deletes the patient
+**Use case: UC02 - Adding a patient**
 
-    Use case ends.
+**MSS**
 
-**Extensions**
+**Use case: UC03 - Listing all patient**
 
-* 2a. The list is empty.
+**MSS**
 
-  Use case ends.
+**Use case: UC04 - Editing a patient**
 
-* 3a. The given id is invalid.
+**MSS**
 
-    * 3a1. MedBook shows an error message.
+**Use case: UC05 - Locating a specific patient**
 
-      Use case resumes at step 2.
+**MSS**
 
-**Use case: Pin a patient**
+**Use case: UC06 - Delete a patient**
 
 **MSS**
 
-1.  User requests to list patients
-2.  MedBook shows a list of patients
-3.  User requests to pin a specific patient in the list
-4.  MedBook pins the patient
+1. User <u>lists all patients (UC03)</u>
+2. User requests to delete a specific patient in the list
+3. MedBook deletes the patient
 
     Use case ends.
 
 **Extensions**
 
-* 2a. The list is empty.
+* ~~2a. The given id is invalid.
+
+    * 2a1. MedBook shows an error message.~~
 
-  Use case ends.
+      Use case resumes at step 2.
```

Comment:
What does this mean? Might not be needed because there's no prompt to ask for delete again. Meaning it is not a must to do delete command again after the error, hence i think this line can be ommited